Makefile

```diff
@@ -43,3 +50,16 @@ test-swift:
 	swift test
 
 test-all: test-linux test-mac test-ios
+
+SDK_PATH = $(shell xcrun --show-sdk-path)
```

Comment:
the place we install the module map is dependent on what Xcode you have selected (via `xcode-select`)